src/test/java/uk/gov/hmcts/reform/cwrdapi/util/EmailValidatorTest.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.reform.cwrdapi.util;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import uk.gov.hmcts.reform.cwrdapi.config.EmailDomainPropertyInitiator;
+
+import javax.validation.ConstraintValidatorContext;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+class EmailValidatorTest {
+    @Mock
+    ConstraintValidatorContext context;
+    @Mock
+    ValidateEmail validateEmail;
+    @InjectMocks
+    private EmailValidator emailValidator;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+        context = mock(ConstraintValidatorContext.class);
+        emailValidator = new EmailValidator();
+        EmailDomainPropertyInitiator.emailDomains = "justice.gov.uk,DWP.GOV.UK,hmrc.gov.uk";
+        validateEmail = mock(ValidateEmail.class);
+        emailValidator.initialize(validateEmail);
+    }
+
+    @Test
+    void testInValid_DomainNotSet_Expected_False() {
+        //if emailDomainList is not set -> expected false
+        var emailId = "vilas.shelke@justice._gov.uk";
+        emailValidator.initialize(validateEmail);
+        assertThat(emailValidator.isValid(emailId, context)).isFalse();
+    }
+
+
+    @Test
+    void testInValid_EmailNotSet_Expected_False() {
+        //if email is empty or null it should be false
+        assertThat(emailValidator.isValid(null, context)).isFalse();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"@justice.gov.uk", "vilas.shelke 123@justice.gov.uk", "vilas.shelke@gmail.com"})
+    void testInValid_EmailWithoutFirstName_Expected_False() {
```

Comment:
parameters not used in test case

Reply:
@rajkatla-hmcts - this was sonar violations. As per sonar practice it fixed. In ValueSource parameters are applied.

Reply:
@vilasshelke-hmcts which one sonar violation ? if you are not using value from source then what is the point of making it as parameterized test ?

Reply:
@rajkatla-hmcts -  @ValueSource(strings = {"@justice.gov.uk", "vilas.shelke 123@justice.gov.uk", "vilas.shelke@gmail.com"}) these are the values

Reply:
@rajkatla-hmcts - modified test case